sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -78,7 +78,7 @@ def _on_attach(self, source, target, properties, error):  # pylint: disable=unus
             self._session_id = session_filter.decode(self._config.encoding)
             self._session._session_id = self._session_id
 
-    def _can_run(self):
+    def _check_session(self):
```

Comment:
Nit: Symmetric naming with _check_live seems prudent, (really liked your rename of that BTW), maybe _check_session_live?

Reply:
_check_session_live is a better name

Reply:
Finally I changed to _check_live and kept it with non-session receiver